src/components/accessories/patientOPD/patientOPDForm/PatientOPDForm.tsx

```diff
@@ -318,20 +391,78 @@ const PatientOPDForm: FunctionComponent<TProps> = ({
               />
             </div>
           </div>
+          <div className="row start-sm center-xs">
+            <div className="patientOpdForm__item fullWidth">
+              <details>
+                <summary>
```

Comment:
This table should be always expanded, at least by default.

Reply:
Done